benchmarks/scripts/common.py

```diff
@@ -61,8 +59,8 @@ def get_compile_function(compiler_alias):
     match compiler_alias:
         case "ucc":
             return ucc_compile
-        case "pytket":
-            return pytket_compile
+        case "pytket-peep":
+            return pytket_peep_compile
```

Comment:
For simplicity, does it make sense to keep this as simply "pytket"? If we are worried about clarity, then we could have part of our documentation that outlines exactly what the pytket/qiskit/cirq compilers are doing.